<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergePopulationPluginComponent</class>
 <widget class="QWidget" name="MergePopulationPluginComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="0">
    <widget class="QCheckBox" name="optGeneratePopFile">
     <property name="text">
      <string>Generate Population file</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="filePopOut"/>
   </item>
   <item row="14" column="2">
    <widget class="QToolButton" name="browsePopOut">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Population file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Stocks</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="componentsResume">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="expand">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qtgui.qrc">
          <normaloff>:/icons/more-horz.png</normaloff>:/icons/more-horz.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="stocksContainer" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QToolButton" name="noStocks">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/noneselect.png</normaloff>:/icons/noneselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="allStocks">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/allselect.png</normaloff>:/icons/allselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="invStocks">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/invselect.png</normaloff>:/icons/invselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <widget class="QTableView" name="stocks"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" rowspan="2" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Size Groups</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="sizeResume">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="expandSizes">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qtgui.qrc">
          <normaloff>:/icons/more-horz.png</normaloff>:/icons/more-horz.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QWidget" name="sizesContainer" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="allGroups">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/allselect.png</normaloff>:/icons/allselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="noGroups">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/noneselect.png</normaloff>:/icons/noneselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="invGroups">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qtgui.qrc">
             <normaloff>:/icons/invselect.png</normaloff>:/icons/invselect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <widget class="QTableView" name="sizes"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtgui.qrc"/>
 </resources>
 <connections/>
</ui>
